polkadot/xcm/xcm-runtime-api/Cargo.toml

```diff
@@ -1,36 +1,32 @@
 [package]
-name = "xcm-fee-payment-runtime-api"
+name = "xcm-runtime-api"
```

Comment:
Very nit: what about `xcm-runtime-apis`?

Reply:
ok, I will do that with all other comments when https://github.com/paritytech/polkadot-sdk/actions/runs/9679467294/job/26705514558?pr=4857 is finished

Reply:
done: https://github.com/paritytech/polkadot-sdk/pull/4857/commits/540433c884d9c66ae5abfdd9a5d843510a0d8233